lib/sucker_punch.rb

```diff
@@ -16,6 +18,18 @@ def self.logger=(logger)
   def self.exception_handler(&block)
     Celluloid.exception_handler(&block)
   end
+
+  def self.clear_actors
```

Comment:
It's be nice to extract the getting of the relevant Celluloid actors to the Queue class, since what we're really asking is, "give me all the queues in the system". Something like:

```ruby
def self.clear_actors
  SuckerPunch::Queue.all.each { |q| q.delete }
end
```